docker/jvm/jsa_launch

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+KAFKA_CLUSTER_ID="$(opt/kafka/bin/kafka-storage.sh random-uuid)"
+opt/kafka/bin/kafka-storage.sh format -t $KAFKA_CLUSTER_ID -c opt/kafka/config/kraft/server.properties
+KAFKA_JVM_PERFORMANCE_OPTS="-XX:ArchiveClassesAtExit=kafka.jsa" opt/kafka/bin/kafka-server-start.sh opt/kafka/config/kraft/server.properties &
+PIDS=$!
+
+sleep 10
+echo "test" | opt/kafka/bin/kafka-console-producer.sh --topic test-topic --bootstrap-server localhost:9092
```

Comment:
If there is an error at any step while creating JSA (e.g. unable to produce/consume) the JSA would have less than optimal information. We should bubble up the failure by failing the build so that this is identified rather than being suppressed.

Reply:
- Valid point. Made the changes to reflect the same
- Creating the topic explicitly now